src/main/java/bio/terra/janitor/app/configuration/TrackResourcePubsubConfiguration.java

```diff
@@ -0,0 +1,43 @@
+package bio.terra.janitor.app.configuration;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.stereotype.Component;
+
+/** Pub/sub Configurations for tracking resource. */
+@Component
+@Configuration
+@EnableConfigurationProperties
+@ConfigurationProperties(prefix = "pubsub.track-resource")
+public class TrackResourcePubsubConfiguration {
+  private boolean pubsubEnabled;
```

Comment:
nit: maybe just `enabled`? I don't think there's any part that's not enabled if this field is false.